test/JDBC/upgrade/13_6/schedule

```diff
@@ -265,3 +265,4 @@ BABEL-3000-dep
 BABEL-3556-before-14_6
 BABEL-3010-before-14_6
 BABEL-sp_helpdb
+BABEL-2795
```

Comment:
I wonder why we did not add this test in 13_4, 13_7 and 13_8 schedule files.

Reply:
added